xrootd.sh

```diff
@@ -72,32 +72,51 @@ cmake "$BUILDDIR"                                                     \
       -DCMAKE_BUILD_TYPE=RelWithDebInfo                               \
       ${OPENSSL_ROOT:+-DOPENSSL_ROOT_DIR=$OPENSSL_ROOT}               \
       ${ZLIB_ROOT:+-DZLIB_ROOT=$ZLIB_ROOT}                            \
-      -DXROOTD_PYBUILD_ENV='CC=c++ CFLAGS=\"-std=c++17\"'             \
-      -DPIP_OPTIONS='--force-reinstall --ignore-installed'            \
+      ${XROOTD_PYTHON:+-DENABLE_PYTHON=ON}                            \
+      ${XROOTD_PYTHON:+-DPYTHON_EXECUTABLE=$PYTHON_EXECUTABLE}        \
+      ${XROOTD_PYTHON:+-DXROOTD_PYBUILD_ENV='CC=c++ CFLAGS=\"-std=c++17\"'}       \
+      ${XROOTD_PYTHON:+-DPIP_OPTIONS='--force-reinstall --ignore-installed -v'}   \
       -DCMAKE_CXX_FLAGS_RELWITHDEBINFO="-Wno-error"
 
 cmake --build . -- ${JOBS:+-j$JOBS} install
 popd
 
-if [[ x"$XROOTD_PYTHON" == x"True" ]];
-then
-  pushd $INSTALLROOT
+if [[ x"$XROOTD_PYTHON" == x"True" ]]; then
+    pushd ${INSTALLROOT}
+
+    # there are cases where python bindings are installed as relative to INSTALLROOT
+    if [[ -d local/lib64 ]]; then
+        [[ -d local/lib64/python${PYTHON_VER} ]] && mv -f local/lib64/python${PYTHON_VER} lib/
+    fi
+    if [[ -d local/lib ]]; then
+        [[ -d local/lib/python${PYTHON_VER} ]] && mv -f local/lib/python${PYTHON_VER} lib/
+    fi
+
     pushd lib
-    if [ -d ../lib64 ]; then
+    if [ -d ../lib64/python${PYTHON_VER} ]; then
       ln -s ../lib64/python${PYTHON_VER} python
-    else
+    elif [[ -d python${PYTHON_VER} ]]; then
       ln -s python${PYTHON_VER} python
     fi
-    popd
-  popd
+    [[ ! -e python ]] && echo "NO PYTHON SYMLINK CREATED in: $(pwd -P)"
+    popd  # get back from lib
+
+    popd  # get back from INSTALLROOT
+
   case $ARCHITECTURE in
-    osx*)
-      find $INSTALLROOT/lib/python/ -name "*.so" -exec install_name_tool -add_rpath ${INSTALLROOT}/lib {} \;
-      find $INSTALLROOT/lib/ -name "*.dylib" -exec install_name_tool -add_rpath ${INSTALLROOT}/lib {} \;
-    ;;
+      osx*)
+        find $INSTALLROOT/lib/python/ -name "*.so" -exec install_name_tool -add_rpath ${INSTALLROOT}/lib {} \;
+        find $INSTALLROOT/lib/ -name "*.dylib" -exec install_name_tool -add_rpath ${INSTALLROOT}/lib {} \;
+      ;;
   esac
-fi
 
+    # Print found XRootD python bindings
+    # just run the the command as this is under "bash -e"
+    echo -ne ">>>>>>   Found XRootD python bindings: "
+    LD_LIBRARY_PATH="$INSTALLROOT/lib${LD_LIBRARY_PATH:+:}$LD_LIBRARY_PATH" PYTHONPATH="$INSTALLROOT/lib/python/site-packages${PYTHONPATH:+:}$PYTHONPATH" ${PYTHON_EXECUTABLE} -c 'from XRootD import client as xrd_client;print(f"{xrd_client.__version__}\n{xrd_client.__file__}");'
```

Comment:
In general, please avoid such long lines.

Reply:
Sure, sorry about that! Would you prefer to fix it now, or given the time for the tests to run just let the tests finish, merge if green, and make the line splitting in a new PR? (maybe in the next one with libxml2?)

Reply:
Yes, that's fine. No need to make a PR just for that.